### DISCUSSION TITLE
Do you have a Discord or Gitter.IM channel?

### DISCUSSION THREAD

**Discussion by rvdbreemen** [bxparks/AceTime#74] [General]
Hi @bxparks 
I like you library development, planning to use it in my project, instead of ezTime.h 
Do you have a online chat setup yet?

Just wondering... 
Thanks,
Robert

**Comment by bxparks** [comment]
I don't have a Discord or Gitter account. This GitHub Discussion is probably the best way to reach me for support questions. Use GitHub Issues for bug reports. Email me directly for PII or sensitive information.
